processing/src/main/java/org/apache/druid/query/aggregation/JavaScriptAggregatorFactory.java

```diff
@@ -285,14 +291,17 @@ public String toString()
    */
   private void checkAndCompileScript()
```

Comment:
Please make it to return a value. Rename to getCompiledScript(). `@EnsuresNonNull("compiledScript")`